content/applications/services/fsm/default_warehouse.rst

```diff
@@ -0,0 +1,40 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
+
+With this feature configured, the products in sales orders created during field interventions are
+always pulled from the same warehouse, keeping the inventory accurate.
+
+.. seealso::
+   :doc:`../../inventory_and_mrp/inventory`
+
+Prerequisites
+=============
+
+To set up a user default warehouse, configure the following settings first:
+
+ #. Open the **Inventory** app, then go to :menuselection:`Configuration --> Settings`. Activate
+    **Storage Locations** and **Multi-Step Routes**.
```

Comment:
How about:

In order to set up a user default warehouse, first go to :menuselection:`Inventory --> Configuration --> Settings` , here activate the :guilabel:Storage Locations` and :guilabel:`Multi-Step Routes`.